UPGRADING

```diff
@@ -429,6 +429,7 @@ Other language changes
 Standard library changes
 ========================
 
+  . substr() now returns an empty string instead of FALSE when the truncation happens on boundaries.
```

Comment:
@staabm I would keep the note in UPGRADING rather terse. One can always look up details in the manual (which also has to be updated in this regard).